sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -67,6 +67,8 @@
 
     raise FeastExtrasDependencyImportError("gcp", str(e))
 
+logger = logging.getLogger(__name__)
```

Comment:
Just a sanity check - is `logging.basicConfig` set up somewhere? If not, I think the logs won't be displayed 😁

Reply:
SInce we're using this in `vmip-feast-materialization-bridge` it should be fine.